### PR TITLE
Update cdp endpoint

### PR DESCRIPTION
**Pull request by imnutz**

### CODE REVIEW

File: lib/plugins/personalization.js [hunk 1]
```diff
@@ -12,11 +12,11 @@ var _ = require('../utils/lodash')
...
- *    Region    cdpHost                       host
- *    AWS East  cdp.in.treasuredata.com       in.treasuredata.com
- *    AWS Tokyo cdp-tokyo.in.treasuredata.com tokyo.in.treasuredata.com
- *    AWS EU    cdp-eu01.in.treasuredata.com  eu01.in.treasuredata.com
- *    IDCF      cdp-idcf.in.treasuredata.com  idcf.in.treasuredata.com
+ *    Region     cdpHost                       host
+ *    AWS East   cdp.in.treasuredata.com       in.treasuredata.com
+ *    AWS Tokyo  cdp-tokyo.in.treasuredata.com tokyo.in.treasuredata.com
+ *    AWS EU     cdp-eu01.in.treasuredata.com  eu01.in.treasuredata.com
+ *    AWS Korea  cdp-ap02.in.treasturedata.com ap02.in.treasuredata.com
```
Comment (tung-vu-td) [inline lib/plugins/personalization.js:19]: We should use official name: `AWS Asia Pacific (Seoul)`

Comment (imnutz) [reply lib/plugins/personalization.js]: 👍

**Comment by tung-vu-td** [comment]
LGTM

**Comment by imnutz** [comment]
Hi @ylzen ,
Could you review this for Data Exfiltrations?

**Pull request merged**